float32 pwm1_freq
float32 pwm1_value
float32 pwm2_freq
float32 pwm2_value
float32 pwm3_freq
float32 pwm3_value
float32 pwm4_freq
float32 pwm4_value
float32 pwm5_freq
float32 pwm5_value
float32 pwm6_freq
float32 pwm6_value
---
uint8 result


